### PR TITLE
Add/641 naked browser launch opts

### LINKED ISSUE #641: Support naked browser launch options
The following script code results in `GoError: parsing launch options: LaunchOptions does not exist in the runtime`. 

```js
const browser = chromium.launch();
```

So we have to provide an empty launch options object as follows:

```js
const browser = chromium.launch({});
```

A better way is to support passing the options as _naked_ in the first example above.

### PR DESCRIPTION
**Pull request by inancgumus**
Fixes #641.

### CODE REVIEW

File: chromium/browser_type.go [hunk 1]
```diff
@@ -124,16 +124,15 @@ func (b *BrowserType) initContext() context.Context {
 
 // Launch allocates a new Chrome browser process and returns a new api.Browser value,
 // which can be used for controlling the Chrome browser.
-func (b *BrowserType) Launch(opts goja.Value) api.Browser {
+func (b *BrowserType) Launch(opts ...goja.Value) api.Browser {
```
Comment (ankur22) [inline chromium/browser_type.go:127]: Why does it need to be done in this way?

For example `browser.newContext()` can be used without supplying any options and it only takes a single `goga.Value` argument:

```go
func (b *Browser) NewContext(opts goja.Value) api.BrowserContext
```

File: examples/launch_naked.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import exec from 'k6/execution';
```
Comment (ankur22) [inline examples/launch_naked.js:1]: Nice, this is good 👍

File: common/browser_options_test.go [hunk 3]
```diff
@@ -262,4 +263,41 @@ func TestBrowserLaunchOptionsParse(t *testing.T) {
 			}
 		})
 	}
+
```
Comment (ankur22) [inline common/browser_options_test.go:266]: Thanks for the test too 👍

File: common/browser_options_test.go [hunk 4]
```diff
@@ -262,4 +263,41 @@ func TestBrowserLaunchOptionsParse(t *testing.T) {
...
+	// will be passed to NewLaunchOptions to set onCloud to false.
+	const onCloud = false
```
Comment (ankur22) [inline common/browser_options_test.go:268]: Very minor, so you can ignore this suggestion.

```suggestion
	const notOnCloud = false
```

I think this reads better when it's used:

```go
wantOpts: NewLaunchOptions(notOnCloud),
```

vs

```go
wantOpts: NewLaunchOptions(onCloud),
```

Comment (ankur22) [reply common/browser_options_test.go:268]: Or maybe set `onCloud` to `true`, and then:

```go
wantOpts: NewLaunchOptions(!onCloud),
```

File: chromium/browser_type.go (hunk 1 above)
Comment (ankur22) [reply chromium/browser_type.go:127]: The browser context parse returns `ni`l if `opts` is `nil`:

```go
if opts != nil && !goja.IsUndefined(opts) && !goja.IsNull(opts) {
   ...
}
return nil
```

Couldn't we do something similar?

**Review by ankur22 — CHANGES_REQUESTED** [review]
I'm a little confused as to why we need to use a variadic function.

The tests are a great addition 👍

File: chromium/browser_type.go (hunk 1 above)
Comment (inancgumus) [reply chromium/browser_type.go:127]: That's a very good point 👍  I'll check it out, thanks.

**Comment by inancgumus** [comment]
You're right. IDK how I missed that 🤦 Thanks, @ankur22 🙇 I've changed the signature to the original and updated the tests. I only left-shifted the logger as I wanted to clarify that `context` and `logger` are peripherals, and we're interested in the options.

**Review by ankur22 — APPROVED** [review]
Looks good, thanks for this!

**Pull request merged**